src/library_syscall.js

```diff
@@ -126,7 +126,7 @@ var SyscallsLibrary = {
 
   _mmap_js__deps: ['$SYSCALLS',
 #if FILESYSTEM && SYSCALLS_REQUIRE_FILESYSTEM
-    '$FS',
+    '$FS', 'emscripten_builtin_memalign',
```

Comment:
I guess this is used from `$FS`, but we don't want to add it there for all users of it? If so a comment on complex deps like these might be good.